### PR TITLE
Adds SetViewportAction

### PR DESCRIPTION
**Pull request by planger**
https://github.com/eclipse-glsp/glsp/issues/179

### CODE REVIEW

File: plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/actions/SetViewportAction.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/********************************************************************************
+ * Copyright (c) 2020 EclipseSource and others.
```
Comment (tortmayr) [inline plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/actions/SetViewportAction.java:2]:
```suggestion
 * Copyright (c) 2021 EclipseSource and others.
```

File: plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/utils/Viewport.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/********************************************************************************
+ * Copyright (c) 2020 EclipseSource and others.
```
Comment (tortmayr) [inline plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/utils/Viewport.java:2]:
```suggestion
 * Copyright (c) 2021 EclipseSource and others.
```

**Review by tortmayr — CHANGES_REQUESTED** [review]
LGTM but the copyright headers need to be updated to 2021

**Comment by planger** [comment]
Thanks! Nicely spotted!
I transferred the copyright from the past into the presence, may the code serve us in the future :)

**Review by tortmayr — APPROVED** [review]

**Pull request merged**